### PR TITLE
Ceph cluster create job is not updated in case of failure

### PR DESCRIPTION
**Pull request by nthomas-redhat**
tendrl-bug-id: Tendrl/commons#599

Signed-off-by: Nishanth Thomas <nthomas@redhat.com>

### CODE REVIEW

File: tendrl/commons/flows/create_cluster/ceph_help.py [hunk 1]
```diff
@@ -222,12 +222,10 @@ def wait_for_task(task_id):
                 if resp["succeeded"]:
                     return
                 else:
-                    stderr = resp.get("stderr", "ceph-installer task_id %s failed and did not complete" % task_id)
-                    stdout = resp.get("stdout", "")
-                    raise FlowExecutionFailedError(dict(ceph_installer_task_id=task_id, ceph_installer_task_stdout=stdout,
-                                        ceph_installer_task_stderr=stderr))
+                    stderr = resp.get("stderr", "")
+                    raise FlowExecutionFailedError("ceph-installer task_id %s \
```
Comment (shtripat) [inline tendrl/commons/flows/create_cluster/ceph_help.py]: Exception was being raised earlier also. How come it was not working then?

**Comment by r0h4n** [comment]
If there's a failure in the job consumer thread, we need to fix it there. Ill send a fix, please test with that.

File: tendrl/commons/flows/create_cluster/ceph_help.py (hunk 1 above)
Comment (shtripat) [inline tendrl/commons/flows/create_cluster/ceph_help.py]: Ah got the answer looking at the issue description. I am ok with this change.

May be small suggestion

```
raise FlowExecutionFailedError(
    "ceph-installer task_id %s failed and didnt complete. Error: %s" %
    (task_id, stderr)
)
```

**Comment by r0h4n** [comment]
Closing this PR, this is not really a fix of the original problem described here https://github.com/Tendrl/commons/issues/599#issuecomment-308930145

Please use below fix and test:
https://github.com/Tendrl/commons/commit/3c477c1c33c297cf7f8e76eb94e12b3131831a43

**Pull request closed without merging**